packages/core/src/Stamp/Stamp.stories.js

```diff
@@ -87,3 +87,67 @@ storiesOf('Stamp', module)
       </Stamp>
     </div>
   ))
+  .add('Pass an array of sizes', () => (
+    <div>
+      <Stamp
+        borderColor='blue'
+        bg='blue'
+        color='white'
+        mr={2}
+        size={['medium', null, null, null, null, null]}
+      >
+        <PinIcon />
+        <Text>Medium at xs</Text>
```

Comment:
I checked the updated storybook in Chromatic, which works great, but can we change these to `Large at xyz`? That'd make it a bit easier to see the differences.